install-moveit2/source/index.markdown

```diff
@@ -77,7 +79,7 @@ Download the repository and install any dependencies:
 
     git clone https://github.com/ros-planning/moveit2.git
     vcs import < moveit2/moveit2.repos
-    rosdep install -r --from-paths . --ignore-src --rosdistro foxy -y
+    rosdep install -r --from-paths . --ignore-src --rosdistro $ROS_DISTRO -y
```

Comment:
This won't be available yet, unless they manually define or source it, right? Please address.

Reply:
the instructions explicitly mention that ROS needs to be sourced in the beginning